swagauth/settings.py

```diff
@@ -152,15 +152,34 @@
 ]
 
 SWAGAUTH_SETTINGS = {
+    'bitbucket': {
+        'APP': {
+            'key': env.str('BITBUCKET_SECRET'),
+            'secret': env.str('BITBUCKET_KEY'),
+        },
+        'SCOPE': {
+
```

Comment:
Scope is not specified. This should include `repository`. Also it should be a list, but not dict.

Reply:
oh, yes it`s my mistake